great_expectations/data_context/util.py

```diff
@@ -124,8 +124,7 @@ def instantiate_class_from_config(config, runtime_environment, config_defaults=N
             "Couldn't instantiate class: {} with config: \n\t{}\n \n".format(
                 class_name, format_dict_for_error_message(config_with_defaults)
             )
-            + str(e)
-        )
+        ) from e
```

Comment:
These `from e` cleanups will give us the backtrace of the causing exception in addition to the exception message.